### PR TITLE
INT-4183 - Provide the ability to ingest a kubernetes secret for jvm …

### PR DESCRIPTION
**Pull request by kellyrob99**
…truststore used by IQ server

- add as simple as possible an example of configuring the truststore(s) for the IQ server

### CODE REVIEW

File: charts/nexus-iq/values.yaml [hunk 1]
```diff
@@ -19,9 +19,27 @@ iq:
   # add this line with this file path and the `licenseSecret` above to autoconfigure licensing
   # licenseFile: /etc/nexus-iq-license/license_lic
   extraLabels:
+  # add the following two lines to mount a secrets volume within the container at the specified location  
+  secretName: #secret-jks
+  secretMountName: #/etc/secret-volume
```
Comment (collinpeters) [inline charts/nexus-iq/values.yaml:24]: Is the `#` supposed to be there or at the front of the line?

File: charts/nexus-iq/README.md [hunk 2]
```diff
@@ -106,3 +108,18 @@ when you first enter the GUI.
...
+```shell
+kubectl create secret generic secret-jks 
+	--from-file=truststore.jks=./truststore.jks 
+	--from-file=keystore.jks=./keystore.jks 
+	--from-literal='keystorePassword=password' 
+	--from-literal='truststorePassword=password'
+```
+
+
+
```
Comment (collinpeters) [inline charts/nexus-iq/README.md:125]: Extra lines?

**Review by collinpeters — APPROVED** [review]
Wow, what an obtuse syntax. I'm no expert (first time seeing these) but just a couple of formatting questions I think. Otherwise +1

**Comment by kellyrob99** [comment]
> Wow, what an obtuse syntax. I'm no expert (first time seeing these) but just a couple of formatting questions I think. Otherwise +1

Indeed, been awhile since I've seen this style too, but it works :) https://helm.sh/docs/chart_template_guide/function_list/#and

File: charts/nexus-iq/values.yaml (hunk 1 above)
Comment (kellyrob99) [reply charts/nexus-iq/values.yaml:24]: > Is the `#` supposed to be there or at the front of the line?

They are part of the contract but deliberately left empty. Could just as easily comment out the whole line?

Comment (jflinchbaugh) [reply charts/nexus-iq/values.yaml:24]: commenting the whole line would be more consistent with other commented values in this file.

**Review by jflinchbaugh — APPROVED** [review]
+1

**Pull request merged**